dotcms-integration/src/test/java/com/dotmarketing/business/VersionableAPITest.java

```diff
@@ -586,4 +594,76 @@ public void test_is_working_with_persisted_in_any_lang_contentlet() throws DotDa
 
 		Assert.assertTrue(APILocator.getVersionableAPI().hasWorkingVersionInAnyOtherLanguage(myContentletIt, languageIt.getId()));
 	}
+
+	private static Debouncer getDebouncer() throws NoSuchFieldException, IllegalAccessException {
```

Comment:
if we use a separate compoenent we wont need to access fields using refections for testing